fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -597,6 +784,9 @@ class Teleop
       components_.push_back(c);
     }
 
+    c.reset(new ArmTeleop("arm", nh));
+    components_.push_back(c);
```

Comment:
these lines should be in the "if (is_fetch)" above -- we don't want arm teleop on freights